normalizer/src/main/scala/tech/sourced/berserker/normalizer/Main.scala

```diff
@@ -0,0 +1,33 @@
+package tech.sourced.berserker.normalizer
+
+import org.apache.spark.sql.SparkSession
+import tech.sourced.berserker.normalizer.model.Schema
+import tech.sourced.berserker.normalizer.service.ExtractorService
+
+object Main extends App {
+  // TODO parametrize
+  var masterUrl = "local[2]"
+  var appName = "normalizer"
+  var grpcHost = "localhost"
+  var grpcPort = 8888
+  var grpcPlainText = true
+  var parquetCompression = "none"
+  var parquetMode = "overwrite"
+  var parquetFilename = "parquet"
+
+  val spark = SparkSession.builder()
+    .appName(appName)
+    .master(masterUrl)
+    .getOrCreate()
+
+  // Start gRPC connection
+  val extractorService = ExtractorService(grpcHost, grpcPort, grpcPlainText)
+
+  val dataRDD = spark.sparkContext.parallelize(extractorService.getRepositoriesData)
```

Comment:
yes, because is the first approach, we are processing all the repositories in the extractor side. I would prefer to query the database and ask to the extractor for repositories one by one. You can see the method in the extractor, to get one repository at a time.